Makefile

```diff
@@ -8,7 +8,7 @@ PKG=github.com/uselagoon/build-deploy-tool
 LDFLAGS=-w -s -X ${PKG}/cmd.bdtVersion=${VERSION} -X ${PKG}/cmd.shortCommit=${SHORTCOMMIT} -X ${PKG}/cmd.bdtBuild=${BUILD} -X "${PKG}/cmd.goVersion=${GO_VER}"
 
 test: fmt vet
-	go test ./...
+	go clean -testcache && go test -v ./...
```

Comment:
I only made this change to verify that the test was failing randomly and that I wasn't crazy. I don't actually know if it's a good idea to keep?

Reply:
Otherwise it LGTM

Reply:
It is fine, it won't impact CI tests because they will all be fresh cache anyway. For local testing it is also fine, so I left it in